hack/csv-generator.go

```diff
@@ -259,6 +337,8 @@ func marshallObject(obj interface{}, writer io.Writer) error {
 		unstructured.SetNestedSlice(r.Object, deployments, "spec", "install", "spec", "deployments")
 	}
 
+	unstructured.SetNestedSlice(r.Object, relatedImages, "spec", "relatedImages")
```

Comment:
Is the `spec.relatedImages` part of the CSV? I cannot find it in the API definition:
https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/clusterserviceversion_types.go#L268

Reply:
@nunnatsa Nahshon, when we spoke about this,  I remember `relatedImages` was supposed to be a part of the CSV spec, if not, where should this data be present?

Reply:
Thsis in not implemented in OLM and so this is not part of the API. Please see the HCO implementation here: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/tools/csv-merger/csv-merger.go#L55..L66

Reply:
OLM doesn't consume it, but `oc adm catalog mirror` does:
https://docs.openshift.com/container-platform/4.6/operators/admin/olm-restricted-networks.html#olm-mirror-catalog_olm-restricted-networks

Reply:
Ok, then the code is good.
Do the `relatedImages` need to specify digest, or can they also use tag?

Reply:
> Ok, then the code is good.
> Do the `relatedImages` need to specify digest, or can they also use tag?

In HCO, We assumes digests. Please see the code of our `digester` here for reference: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/tools/digester/main.go